cypress/e2e/specialFeatures/cookieBanner/testsForAMPOnly.js

```diff
@@ -17,98 +17,174 @@ export default ({ service, variant, pageType, path }) => {
       retries: 3,
       testIsolation: true,
     },
-    () => {
-      it('should have a privacy & cookie banner, which disappears once "accepted" ', () => {
-        getPrivacyBanner(service, variant).should('be.visible');
+    function test() {
```

Comment:
Switched this to a fat function, to reduce repeated code for the toggles

Reply:
Would it be worth extracting out and adding a `beforeEach` to reduce repetition on the banner visibility checks? Something like:
```
beforeEach(() => {
  if (Cypress.env('toggles')?.privacyPolicy?.enabled) {
    getPrivacyBanner(service, variant).should('be.visible');
  } else {
    getCookieBannerAmp(service, variant).should('be.visible');
  }
});

```

Or is this an example of where a WET approach might be better? Just curious about causes of flakes in E2E's if the banners don't load quickly

Reply:
I would say a wet approach would be better in my opinion, as it makes it explicit for what the order of checks are for each individual test